src/pages/landing-page/LandingPage.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./landing-page.css";
+import cardImage from "../../assets/images/hero.jpg";
+import { Footer, Navigation, Category } from "../../components";
+import { categoryDataList } from "../../data";
+
+const LandingPage = () => {
+  return (
+    <>
+      <Navigation />
+      <main>
+        <section id="hero" className="hero">
+          <div className="hero-wrapper">
+            <div className="hero-text-container">
+              <h1>YOGA ACCESSORIES</h1>
+              <p className="text-xl text-gray">UP TO 30% OFF</p>
+              <p>
+                Top deals on yoga mats, cotton shirts and pants. Find your best
+                size at affordable price. No cost EMI. Complete Protection.
+              </p>
+              <a className="btn btn-primary" href="#category">
+                Explore Now
+              </a>
+            </div>
+            <div className="hero-img-container">
+              <img className="image-resp" src={cardImage} alt="hero" />
+            </div>
+          </div>
+        </section>
+        <section id="category" className="m-top-bottom">
+          <h2 className="text-center">CATEGORIES</h2>
+          <div className="category-container">
+            {categoryDataList.map((item) => (
+              <Category key={item.id} categoryData={item} />
+            ))}
+          </div>
+        </section>
+      </main>
+      <Footer />
+    </>
```

Comment:
Suggestion: The JSX for hero section could be a component like <HeroSection/>

Reply:
I only use it here itself, if we use only once then i think there is no need to make it a component. Correct me if i am wrong.